build/LibraryVersions.g.cs

```diff
@@ -19,6 +19,8 @@ public static class LibraryVersion
             {
                 "8.0.0",
                 "8.1.1",
+                "8.1.2",
```

Comment:
I think it will be good enough to just test against `8.0.0` and `8.1.3`

Reply:
Currently, the strategy is to test the library instrumentations against following versions:

its lowest supported, but not vulnerable, version,
one version from every major release,
the latest supported version (defined in [test/Directory.Packages.props](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/test/Directory.Packages.props)),
other specific versions, eg. containing breaking changes for our instrumentations.

From: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/developing.md#integration-tests